kubernetes/samples/charts/ingress-per-domain/README.md

```diff
@@ -52,21 +52,39 @@ voyager:
   # stats port
   statsPort: 30315
 ```
+
+Sample `values.yaml` for the Nginx Ingress:
+```
+type: NGINX
+
+# WLS domain as backend to the load balancer
+wlsDomain:
+  domainUID: domain1
+  clusterName: cluster1
+  managedServerPort: 8001
+
+# Nginx specific values
+nginx:
+  # hostname used by host-routing
+  hostname: domain1.org
+```
+
 ## Uninstalling the chart
 To uninstall and delete the `my-ingress` deployment:
 ```
-$ helm delete --purge my-ingress
+$ helm uninstall my-ingress
 ```
 ## Configuration
 The following table lists the configurable parameters of this chart and their default values.
 
 | Parameter | Description | Default |
 | --- | --- | --- |
-| `type` | Type of Ingress controller. Legal values are `TRAEFIK` or `VOYAGER`. | `TRAEFIK` |
+| `type` | Type of Ingress controller. Legal values are `TRAEFIK` or `VOYAGER` or 'NGINX'. | `TRAEFIK` |
```

Comment:
'NGINX' needs to be code font (surround with back ticks, `NGINX`

Reply:
done